prime-router/src/test/kotlin/fhirengine/translation/hl7/FhirTransformerTests.kt

```diff
@@ -763,4 +769,36 @@ class FhirTransformerTests {
         assertThat(resource.meta.lastUpdated).isEqualTo(bundle.timestamp)
         assertThat(newValue[0].primitiveValue()).isEqualTo("2021-08-09T05:52:00.000-07:00")
     }
+
+    @Test
+    fun `test extending schema overwrite element`() {
+        val actionLogger = ActionLogger()
+        val fhirBundle = File("src/test/resources/fhirengine/engine/valid_data.fhir").readText()
+        val messages = FhirTranscoder.getBundles(fhirBundle, actionLogger)
+        val bundle = messages[0]
+
+        val childSchema = ConfigSchemaReader.readSchemaTreeFromFile(
```

Comment:
The `mergeSchemas` function doesn't actually get called by this function. I'm tempted to make `readSchemaTreeFromFile` a private function and replace it with `fromFile` in all of our tests to prevent this confusion in the future.. but I'm undecided on that. Either way, we at least need to do this:
```suggestion
        val childSchema = ConfigSchemaReader.fromFile(
```

Reply:
It might even be better to use `fhirTransformSchemaFromFile` since that's what we'll use to read fhir transform schemas in the code.